app/src/main/kotlin/com/simplemobiletools/flashlight/activities/BrightDisplayActivity.kt

```diff
@@ -27,40 +45,65 @@ class BrightDisplayActivity : SimpleActivity() {
                 WindowManager.LayoutParams.FLAG_FULLSCREEN
         )
```

Comment:
Can we have these flags migrated to non deprecated versions for >= R

Reply:
Before updating this, I didn't even notice that this activity wasn't properly full screen :smile: 
Good catch!